src/user/user.controller.ts

```diff
@@ -73,8 +68,11 @@ export class UserController {
     return this.userService.update(rut, updateUserDto);
   }
 
-  @Delete(':id')
-  remove(@Param('id') id: string) {
-    return this.userService.remove(+id);
+@Delete('delete/:id')
+  @UseGuards(JwtAuthGuard, RolesGuard)
+  @Roles(Role.ADMIN)
+  async remove(@Param('id') id: number) {
+    return await this.userService.softDelete(id);
```

Comment:
It should be better if you use UserService's method `remove` instead of `softDelete`

Reply:
fixed by https://github.com/alkemyTech/CENCO-MIT-Back-T1/pull/27/commits/5657f8ccd61312b517ada70345cd19d2158d162b